app/src/main/java/com/hamdan/acceleromatorminiproject/data/AccelerationDao.kt

```diff
@@ -0,0 +1,33 @@
+package com.hamdan.acceleromatorminiproject.data
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface AccelerationDao {
+
+    @Insert
+    suspend fun insert(scoreEntity: ScoreEntity)
+
+    @Query(
+        """
+        DELETE FROM ${ScoreEntity.SCORE_TABLE_NAME}
+        """
+    )
+    suspend fun nukeTable()
+
+    @Query(
+        """
+            SELECT * FROM ${ScoreEntity.SCORE_TABLE_NAME} 
+            ORDER BY ${ScoreEntity.ELAPSED_TIME} DESC, 
+            ${ScoreEntity.DATE_ACQUIRED} ASC
+            LIMIT $HIGH_SCORE_LIST_SIZE
```

Comment:
What about the date? what if two values have the same score but the dates are different, which should get grabbed instead of the other?

Reply:
This SQL query will sort them by the elapsed time, and also sort by date acquired

The priority will be as follows: Sort by Score (highest to lowest) -> Sort by Date (lowest to highest)

in other words, the first to score the higher score will be grabbed from the database